Automattic/jetpack
# wp-admin Command Palette: Declare external dependencies

## Proposed changes:

In https://github.com/Automattic/wp-calypso/pull/88193, we're externalizing some dependencies to optimize the bundle of the command palette app, since these dependencies are enqueued by default on all WP sites.

However, a custom plugin could dequeue them. To mitigate this scenario, we're now explicitly declaring the dependencies needed by the command palette so they are properly enqueued if needed.

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Apply these changes to your site
- Follow instructions from https://github.com/Automattic/wp-calypso/pull/88193
- Make sure the command palette is loaded
